packages/slate/src/interfaces/node.ts

```diff
@@ -468,7 +472,7 @@ export const Node = {
     if (Text.isText(node)) {
       return node.text
     } else {
-      let children = node.children as Array<Descendant>
+      const children = node.children as Array<Descendant>
```

Comment:
`Array<x>` is more commonly written as `x[]`

```suggestion
      const children = node.children as Descendant[]
```

Reply:
Tim is totally correct but we should avoid the type assertion, if we can.